.github/CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ Note that your contributions must be your own work and licensed under the same t
 ## Reporting bugs
 
 First, search the issue tracker to see if the bug is already there.
-Please use the [discussion forums](http://dotspatial.codeplex.com/discussions) if you are unsure whether it is a bug in the library or a problem in your code.
+Please use the [discussion forums](https://github.com/DotSpatial/DotSpatial/issues) if you are unsure whether it is a bug in the library or a problem in your code.
```

Comment:
Please change this to discussions. Don't want to have a lot of issues for things that might not be bugs.

Reply:
done